lib/iron_worker_ng/feature/python/merge_requirements.rb

```diff
@@ -0,0 +1,35 @@
+module IronWorkerNG
+  module Feature
+    module Python
+      module MergeRequirements
+        module InstanceMethods
+
+          def merge_requirements(path)
+            feature = IronWorkerNG::Feature::Base.new(self)
+            IronWorkerNG::Fetcher.fetch_to_file(feature.rebase(path)) do |requirements|
```

Comment:
as I understand it merge_pip does the following: 
```
pip install flask>=1.0.0 django>=1.6.0
```
I offer, what if you pass `'-r /path/to/requirements.txt'` instead of `flask>=1.0.0 django>=1.6.0`.
requirements.txt file parsed correctly for me.